src/osiris/pipelines/transformations.py

```diff
@@ -61,11 +61,12 @@ class JoinUniqueEventData(beam_core.DoFn, ABC):
     Takes a list of events and join it with processed events, if such exists, for the particular event time.
     It will only keep unique pairs.
     """
-    def __init__(self, datasets: Dataset, time_resolution: TimeResolution):
+    def __init__(self, datasets: Dataset, time_resolution: TimeResolution, index_columns: list):
```

Comment:
Problemet med dette er, at alle transformationer skal have angivet denne liste. En mulighed er at det bliver optional.
Derudover skal vi på alle transformationer kunne angive denne liste. For den generiske transformation, skal dette gøres via conf.ini. Det skal også implementeres før dette kan merges.

Jeg ser ikke noget problem i det, men vi kan ikke gøre det uden vi har taget stilling til det.

Reply:
```suggestion
    def __init__(self, datasets: Dataset, time_resolution: TimeResolution, index_columns: list=None):
```
Added default option to be non breaking.